mlflow/R/mlflow/.create-test-env.R

```diff
@@ -12,4 +12,4 @@ keras::install_keras(method = "conda", envname = mlflow:::mlflow_conda_env_name(
 # TODO: unpin after we use tensorflow >= 2.4
 reticulate::conda_install("'h5py<3.0.0'", envname = mlflow:::mlflow_conda_env_name(), pip = TRUE)
 reticulate::conda_install("xgboost", envname = mlflow:::mlflow_conda_env_name())
-reticulate::conda_install(paste0("h2o==", packageVersion("h2o")), envname = mlflow:::mlflow_conda_env_name(), pip = TRUE)
+reticulate::conda_install(paste0("h2o=", packageVersion("h2o")), envname = mlflow:::mlflow_conda_env_name(), channel = c("conda-forge"))
```

Comment:
This doesn't seem to work.

Reply:
I'll try to install h2o from  https://h2o-release.s3.amazonaws.com/h2o/latest_stable_R.html